queries/query_builders.go

```diff
@@ -367,60 +367,70 @@ ManualParen:
 			buf.WriteByte(')')
 		case whereKindIn:
 			ln := len(where.args)
-			matches := rgxInClause.FindStringSubmatch(where.clause)
-			// If we can't find any matches attempt a simple replace with 1 group.
-			// Clauses that fit this criteria will not be able to contain ? in their
-			// column name side, however if this case is being hit then the regexp
-			// probably needs adjustment, or the user is passing in invalid clauses.
-			if matches == nil {
-				clause, count := convertInQuestionMarks(q.dialect.UseIndexPlaceholders, where.clause, startAt, 1, ln)
+			// WHERE IN () is invalid sql, so it is difficult to simply run code like:
+			// for _, u := range model.Users(qm.WhereIn("id IN ?",uids...)).AllP(db) {
+			//    ...
+			// }
+			// instead when we see empty IN we produce 1=0 so it can still be chained
+			// with other queries
+			if ln == 0 {
+				buf.WriteString("(1=0)")
```

Comment:
Sorry for taking so long to get to this. I've decided to merge this but this should really have been an early `break` to avoid indenting the remainder of the code. If you change that I'll merge it :)

Reply:
@aarondl sorry for the delay, i missed the message; i pushed the early break